### PR TITLE
enhancement: display relations on history page

### PR DESCRIPTION
**Pull request by remidej**
### What does it do?

- Displays the relations in the frontend using a custom input component, as the normal one does not work for our needs and the data we have
- Displays a warning alert when a relation or a media input is missing
- Fixes some UI issues:
  - apply bg color on the entire page
  - removes the sticky header we didn't want
  - adds some padding at the bottom of the page

### How to test it?

- Open the history for an entry with relations. You should see the relation's main field, or the document id as a fallback. Clicking on the name should take you to the edit view for that entry.
- Add relations to an entry, then delete the entries you just linked (don't just unlink them, go to their own page in the CM and delete them). You should see the "missing content" banner.
- Do the same but for media assets.

### Related issue(s)/PR(s)

This follows #20035 which was about the backend part.

### CODE REVIEW

File: packages/core/content-manager/admin/src/history/components/VersionContent.tsx [hunk 1]
```diff
@@ -2,41 +2,75 @@ import * as React from 'react';
...
+    value: { results, meta },
+  } = useField(props.name);
 
   return (
     <Box>
       <FieldLabel>{props.label}</FieldLabel>
-      {field.value.results.length === 0 ? (
+      {results.length === 0 ? (
         <Typography>No content</Typography>
       ) : (
```
Comment (remidej) [inline packages/core/content-manager/admin/src/history/components/VersionContent.tsx:64]: I'm waiting on @lucasboilly for the empty state design, this is a placeholder

Comment (lucasboilly) [reply packages/core/content-manager/admin/src/history/components/VersionContent.tsx:64]: Done, please find it [here](https://www.figma.com/file/XYJUrJn9i9ius3bOUSL4fn/Content-History?type=design&node-id=1444%3A22532&mode=design&t=RPZeMPo0TUyWrB1I-1) :)

File: packages/core/content-manager/admin/src/history/components/VersionContent.tsx [hunk 2]
```diff
@@ -2,41 +2,82 @@ import * as React from 'react';
...
+              id: 'content-manager.history.content.no-relations',
+              defaultMessage: 'No relations.',
+            })}
+          </StyledAlert>
+        </Box>
       ) : (
-        <Flex direction="column" gap={2} alignItems="stretch">
-          {(field.value.results as Record<string, unknown>[]).map((relationData, index) => {
+        <Flex direction="column" gap={2} marginTop={1} alignItems="stretch">
+          {(results as RelationResult[]).map((relationData) => {
```
Comment (joshuaellis) [inline packages/core/content-manager/admin/src/history/components/VersionContent.tsx:73]: you can type the `useField` hook, it'd probably look like:

```ts
useField<{ results: RelationResult[]; meta: object }>(props.name)
```

**Review by joshuaellis — APPROVED** [review]
Looks great 👍🏼

**Pull request merged**